src/scss/index.scss

```diff
@@ -85,9 +85,11 @@ $color-bg--accept             : $green--offWhite;
 $color-bg--notification       : $yellow--light;
 $color-bg--info               : $blue--offWhite;
 $color-bg--error              : $red--offWhite;
+$color-bg--default            : $blue;
 
 $color-border                 : $grey--lightest;
 $color-border--interactive    : $grey--lighter;
+$color-border--default        : $blue;
```

Comment:
Same here – where will this be used (as may be able to name it a bit better so it's more obvious the difference from `$color-border`

Reply:
Again with this one @ashleynolan

Reply:
And this has this class...

`c-scheduleTypeSwitcher-switch--active`

@ashleynolan